app/views/accountants/_search_form.html.erb

```diff
@@ -1,12 +1,21 @@
 <div id="search_form" class="margin-bottom">
-  <!-- TODO change submit tag to a search glyphicon -->
   <%= bootstrap_form_with url: 'accountant/search', remote: true, layout: :inline do |f| %>
     <%= f.hidden_field :locale, value: params[:locale] %>
     <%= f.text_field :search,
                      placeholder: t('accountants.search_placeholder'),
                      hide_label: true,
-                     class: 'search_form_input accountant' %>
-    <%= f.submit t('common.search'), class: 'btn btn-primary ml-2' %>
+                     class: 'search_form_input accountant mr-4' %>
+
+    <%= f.select :clinic_id,
+                 options_for_select(clinic_options.reject{|x| x.blank?}),
+                 prompt: "[All Clinics]",
```

Comment:
Adding a `prompt` adds an item to the dropdown with empty value. The above search code, then, sees a blank clinic search and ignores it for filtering purposes.

Reply:
I had no idea rails had this! amazing

Reply:
I think is kinda an HTML thing!

Reply:
Let me know if you're fine with that formatting etc. In particular, do I need to handle internationalization? Or does that happen in batches down the line?